redwood-compose/src/commonMain/kotlin/app/cash/redwood/compose/RedwoodComposition.kt

```diff
@@ -71,14 +156,67 @@ public inline fun <F : Widget.Factory<*>, W : Widget<*>> RedwoodComposeNode(
   }
 
   Updater<W>(currentComposer).update()
-  content()
+  _RedwoodComposeContent.Instance.content()
 
   currentComposer.endNode()
 }
 
 /**
  * @suppress For generated code usage only.
  */
-public interface RedwoodApplier<F : Widget.Factory<*>> {
+@Suppress("ClassName") // Hiding from auto-complete.
```

Comment:
I don't think that helps since auto-complete doesn't know anything about that convention. I finished this refactoring and most types become private. The ones which stay public I'm going to put behind an `@OptIn` annotation instead.